src/theory/bv/bv_eager_solver.cpp

```diff
@@ -34,26 +34,32 @@ EagerBitblastSolver::EagerBitblastSolver(TheoryBV* bv)
 
 EagerBitblastSolver::~EagerBitblastSolver() {
   if (d_useAig) {
-    Assert (d_bitblaster == NULL); 
+    Assert (d_bitblaster == NULL);
     delete d_aigBitblaster;
   }
-  else {
-    Assert (d_aigBitblaster == NULL); 
+  else
+  {
+    Assert (d_aigBitblaster == NULL);
     delete d_bitblaster;
   }
 }
 
 void EagerBitblastSolver::turnOffAig() {
-  Assert (d_aigBitblaster == NULL &&
-          d_bitblaster == NULL);
+  Assert (d_aigBitblaster == NULL);
+  Assert (d_bitblaster == NULL);
   d_useAig = false;
 }
 
 void EagerBitblastSolver::initialize() {
   Assert(!isInitialized());
   if (d_useAig) {
+#ifdef CVC4_USE_ABC
```

Comment:
This `#ifdef` is a bit inconsistent in style with the other ones. I like the other style better (where you have the `#ifdef` inside of the `then` block.